internal/controller/paas_controller.go

```diff
@@ -194,23 +193,63 @@ func (r *PaasReconciler) Reconcile(ctx context.Context, req ctrl.Request) (resul
 	paas.Status.Truncate()
 
 	if err := r.ReconcileQuotas(ctx, paas); err != nil {
-		return ctrl.Result{}, errors.Join(err, r.setErrorCondition(ctx, paas, err))
+		err = r.setErrorCondition(ctx, paas, err)
```

Comment:
This is old. Please rebase on upstream.

Reply:
Yes, I reverted previous changes (silly me).
And that indeed resolved my last issue. Thanks.